frontend/src/components/Footer.vue

```diff
@@ -1,15 +1,15 @@
 <template>
   <footer class="footer">
-      <v-container class="pa-0">
+      <v-container class="py-0 px-12">
       <!-- Navbar content -->
         <v-row>
-            <v-btn text id="footer-home" href="https://www.gov.bc.ca/">{{$tc('Home')}}</v-btn>
-            <v-btn text id="footer-about" href="https://www2.gov.bc.ca/gov/content/about-gov-bc-ca">{{$tc('About gov.bc.ca')}}</v-btn>
-            <v-btn text id="footer-disclaimer" href="http://gov.bc.ca/disclaimer/">{{$tc('Disclaimer')}}</v-btn>
-            <v-btn text id="footer-privacy" href="http://gov.bc.ca/privacy/">{{$tc('Privacy')}}</v-btn>
-            <v-btn text id="footer-accessibility" href="http://gov.bc.ca/webaccessibility/">{{$tc('Accessibility')}}</v-btn>
-            <v-btn text id="footer-copyright" href="http://gov.bc.ca/copyright">{{$tc('Copyright')}}</v-btn>
-            <v-btn text href="https://www2.gov.bc.ca/gov/content/home/contact-us">{{$tc('Contact Us')}}</v-btn>
+            <a class="px-3 pt-2" id="footer-home" href="https://www.gov.bc.ca/">{{$tc('Home')}}</a>
```

Comment:
For consistency this should be v-btns that are depressed with the px mx etc as needed

Reply:
done